cli/src/index.ts

```diff
@@ -1,13 +1,17 @@
 #! /usr/bin/env node
 
 import { Option, program } from 'commander';
+import { createRequire } from 'module';
 import { visualizeInstantiation, visualizePattern } from './commands/visualize/visualize.js';
 import { runGenerate } from './commands/generate/generate.js';
 import validate  from './commands/validate/validate.js';
 import { CALM_META_SCHEMA_DIRECTORY } from './consts.js';
 
+const require = createRequire(import.meta.url);
+const { version } = require('../package.json');
```

Comment:
Modern linters will block this from being merged 

![image](https://github.com/user-attachments/assets/fbd372f0-95ea-4957-916e-15797765990a)

Can I suggest adding `resolveJsonModule: true` to `tsconfig.json` and then changing this to `import { version } from '../package.json';`

Reply:
Updated per suggestion